sdk/monitor/Azure.Monitor.Ingestion/tests/SessionRecords/MonitorIngestionLiveTest/ConcurrencyMultiThreadAsync.json

```diff
@@ -1,180 +1,180 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://98c96978-a857-58f1-a299-7d216d4b043a-p2j8.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-37a02bb9a8ba4f97a80716652c78e678/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
+      "RequestUri": "https://86923540-75e9-5143-84b6-7535316a68dc-lipn.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-ba536bf3cee94b1db8d04b30ee3f2e80/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
       "RequestMethod": "POST",
       "RequestHeaders": {
         "Accept": "application/json",
         "Authorization": "Sanitized",
         "Content-Encoding": "gzip",
-        "Content-Length": "85",
+        "Content-Length": "88",
         "Content-Type": "application/json",
-        "traceparent": "00-0feae899d9b5ac4f8bb76518014d8cfe-a1bf7b798824644e-00",
-        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20221012.1 (.NET Core 3.1.30; Microsoft Windows 10.0.22621)",
-        "x-ms-client-request-id": "b37c02315c1b49105bd815dbc0da4c3b",
+        "traceparent": "00-84024e655bda424a8663ab11d00b1efd-ce07df2b0dc90d49-00",
+        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20221024.1 (.NET Core 3.1.30; Microsoft Windows 10.0.22621)",
+        "x-ms-client-request-id": "abdc147dba7aeba7b74c902c74a99d4d",
         "x-ms-return-client-request-id": "true"
       },
-      "RequestBody": "[{\u0022Time\u0022:\u00222022-10-13T14:10:38.720123\u0022,\u0022Computer\u0022:\u0022Computer4\u0022,\u0022AdditionalContext\u0022:4}]]",
+      "RequestBody": "[[{\u0022Time\u0022:\u00222022-10-24T17:04:30.6669136\u0022,\u0022Computer\u0022:\u0022Computer0\u0022,\u0022AdditionalContext\u0022:0}],]",
```

Comment:
Hmm this is strange - it seems we now have nested arrays? Did something change in the test?

Reply:
Removed test recordings from this PR, will take a look in https://github.com/Azure/azure-sdk-for-net/issues/31767